emr.tf

```diff
@@ -0,0 +1,107 @@
+resource "aws_emr_cluster" "cluster" {
+  name                              = local.emr_cluster_name
+  release_label                     = var.emr_release_label
+  applications                      = var.emr_applications
+  termination_protection            = var.termination_protection
+  keep_job_flow_alive_when_no_steps = var.keep_flow_alive
+  //TODO The below is need for transparent encryption/decryption when insecure DKS is set up
+  //security_configuration            = aws_emr_security_configuration.emrfs_em.id
+  service_role                      = aws_iam_role.analytical_dataset_generator.arn
+  //TODO which is the right bucket for logs , publish bucket??
+  log_uri                           = format("s3n://%s/logs/", aws_s3_bucket.published.id)
```

Comment:
This should point at the logstore bucket managed by aws-security-tools. See https://github.ucds.io/dip/aws-security-tools/blob/master/logstore.tf#L85-L89

Reply:
Changed as suggested